daemon/container_operations.go

```diff
@@ -1053,3 +1050,10 @@ func (daemon *Daemon) DeactivateContainerServiceBinding(containerName string) er
 	}
 	return sb.DisableService()
 }
+
+func getNetworkID(name string, endpointSettings *networktypes.EndpointSettings) string {
```

Comment:
Alway try to use `NetworkID` whenever `NetworkID` is available.

Reply:
This name is a bit odd (from just the name, I would expect it to _always_ return an ID), not a big thing because it's not exported, but perhaps we should find a better name for it